internal/service/rds/instance_test.go

```diff
@@ -3579,17 +3579,41 @@ data "aws_rds_orderable_db_instance" "test" {
 `, engine, version, license)
 }
 
+func testAccInstanceConfig_orderableClass_SQLServerEx(engine, version, license string) string {
+	return fmt.Sprintf(`
+data "aws_rds_orderable_db_instance" "test" {
+  engine         = %q
+  engine_version = %q
+  license_model  = %q
```

Comment:
```suggestion
  engine         = %[1]q
  engine_version = %[2]q
  license_model  = %[3]q
```
Best to always include index numbers for multi-line strings so that when someone adds a line between later is doesn't mess it up.

Reply:
Ooo yeah good call 👍